vllm/engine/arg_utils.py

```diff
@@ -336,6 +337,12 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
             'Model Streamer \n'
             '* "bitsandbytes" will load the weights using bitsandbytes '
             'quantization.\n')
+        parser.add_argument("--weights-load-device",
+                            type=str,
+                            default=EngineArgs.weights_load_device,
+                            choices=DEVICE_OPTIONS,
+                            help='Device to which model weights '
+                            'will be loaded.')
```

Comment:
Why do you need to load the weights on CPU? How do they end up on the device? This seems like a feature that will be difficult to support for the many different backends